actors/builtin/miner/miner_test.go

```diff
@@ -848,6 +972,13 @@ func (h *actorHarness) onProvingPeriodCron(rt *mock.Runtime, expectedEnrollment
 		rt.ExpectGetRandomness(crypto.DomainSeparationTag_WindowedPoStDeadlineAssignment, randEpoch, nil, bytes.Repeat([]byte{0}, 32))
 	}
 	// Re-enrollment for next period.
+	reward := big.NewIntUnsigned(1e18)
+	rt.ExpectSend(builtin.RewardActorAddr, builtin.MethodsReward.LastPerEpochReward, nil, big.Zero(), &reward, exitcode.Ok)
+
+	pwrTotal := power.CurrentTotalPowerReturn{
+		QualityAdjPower: big.NewIntUnsigned(1 << 50),
+	}
+	rt.ExpectSend(builtin.StoragePowerActorAddr, builtin.MethodsPower.CurrentTotalPower, nil, big.Zero(), &pwrTotal, exitcode.Ok)
```

Comment:
This whole pattern happens more than once, so break it out into a helper `expectQueryNetworkInfo(qaPower, epochReward)`.